polaris-react/src/components/LegacyTabs/LegacyTabs.stories.tsx

```diff
@@ -87,6 +94,51 @@ export function Default() {
   );
 }
 
+export function WithUrlTabs() {
+  const [selected, setSelected] = useState(0);
+
+  const handleTabChange = useCallback(
+    (selectedTabIndex) => setSelected(selectedTabIndex),
+    [],
+  );
+
+  const tabs = [
+    {
+      id: 'all-customers-2',
+      content: 'All',
+      accessibilityLabel: 'All customers',
+      panelID: 'all-customers-content-2',
```

Comment:
Changing the id on this should help with failing accessibility check
```suggestion
      id: 'all-customers-5',
      content: 'All',
      accessibilityLabel: 'All customers',
      panelID: 'all-customers-content-5',
```